tests/stdlib/tos.nim

```diff
@@ -54,6 +54,16 @@ block fileOperations:
     doAssert not dirExists(dname/sub)
     removeFile(dname/fname)
     removeFile(dname/fname2)
+    when not defined(windows):
+      let brokenSymlink = dname/"D20210101T191320_BROKEN_SYMLINK"
+      let brokenSymlinkDest = "D20210101T191320_I_DO_NOT_EXIST"
```

Comment:
actually docs call this src: `proc createSymlink(src, dest: string)`; maybe `brokenSymlinkSrc` ?